src/event-stream/event-stream.interfaces.ts

```diff
@@ -56,8 +56,13 @@ export interface Event {
   logIndex: string;
   timestamp: string;
   data: any;
-  inputMethod?: string;
-  inputArgs?: Record<string, any>;
+  inputMethod: string;
+  inputArgs: {
+    amount: string;
+    data: string;
+    to?: string;
```

Comment:
Maybe a nitpick, but I think the interface at this level should be generic to any event stream. If you expect particular named args I would cast that in the tokens service.